src/ms-store-badge.d.ts

```diff
@@ -13,6 +13,18 @@ declare class MSStoreBadge extends HTMLElement {
      * The ID of your app.
      */
     productId: string;
+    /**
+     * The optional campaign ID of your app.
+     */
+    cid: string;
+    /**
+      * Indicates whether popup or full mode should be launched.
+      */
+    popup: string;
+    /**
+      * Indicates whether badge should be in dark mode or light mode.
+      */
+    darkMode: string;
```

Comment:
Likewise, this is unintuitive to me. Why is darkMode a string? Is it related to the attribute specification we talked about earlier? Even so, I feel like this should be a boolean, and we set the boolean based on whether the attribute is present.